src/main/java/exercising/learnig/methoding/EndOfGameConditions.java

```diff
@@ -0,0 +1,55 @@
+package exercising.learnig.methoding;
+
+import exercising.learnig.arraysCreating.Paair;
+
+import java.util.List;
+
+public class EndOfGameConditions {
+
+   char e;
+
+    public EndOfGameConditions(char e) {
+        this.e = e;
+    }
+
+    public boolean condition1(List<Paair> paairState){
```

Comment:
to ze masz na stale oznaczone ktore linie wygrywaja a ktore przegrywaja, ok :D (chociaz mozna to obliczyc pobierajac za kazdym razem sasiada punktu i np rekurencyjnie sprawdzac czy juz sa 3 w linii :D 

ale polaczenie tych metod razem z tym ifem ktory to sprawdza juz mnie boli troche :D 

Zaproponuje inne rozwiazanie :D 
klasa Condition z 3 polami Paair. 
Konstruktor moze przyjmowac 3 inty i znak (e) 

I teraz mozesz stworzyc kolejna klase WinningCondition 
ktora bedzie przechowywac liste wszystkich Condition potrzebnych zeby wygrac. 
Potem tylko jakas metoda do sprawdzenia czy ktos wygral i gotowe :D

Reply:
albo nawet lepiej w polach trzymac tylko te 3 inty, 
a x i o podawac tylko dla funkcji ktora ma sprawdzic wynik :D